### PR TITLE
GitHub Actions: NodeJS CI Pipeline Workflow

### LINKED ISSUE #3: setup ci for testing changes

### PR DESCRIPTION
**Pull request by sambacha**
This PR closes #3 

Signed-off-by: sam <sam@freighttrust.com>

### CODE REVIEW

**Comment by epheph** [comment]
@thegostep i haven't used actions before, this look good to you?

**Review by thegostep — CHANGES_REQUESTED** [review]
thanks for the pr 👍 

Let's test against other node versions.

Adding testing will require setting up secrets which I can handle in a separate issue.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: ['12.x']
```
[the remark this answers is not in the document]
Comment (sambacha) [reply .github/workflows/nodejs.yml]: Added 10, 12, and 14 for the CI Pipeline. 

Note: I did not add cache to this workflow as the different versions for NodeJS may produce different `node_modules` dependencies, which can be done by adding each version as a separate job. I figured to keep this workflow rather simple for now.

**Review by thegostep — APPROVED** [review]

**Comment by sambacha** [comment]
💯 - If yall have any sort of developer styling preferences (e.g. Angular style git commits, etc), just let me know! 

Have a great weekend,

Sam

**Pull request merged**